packages/cashier/src/stores/__tests__/on-ramp-store.spec.ts

```diff
@@ -2,31 +2,32 @@ import { waitFor } from '@testing-library/react';
 import OnRampStore from '../on-ramp-store';
 import createBanxaProvider from '../../pages/on-ramp/on-ramp-providers';
 import { configure } from 'mobx';
-import { TRootStore, TWebSocket, TOnRampProvider } from 'Types';
+import { TWebSocket, TOnRampProvider } from '../../types';
+import { mockStore, TStores } from '@deriv/stores';
```

Comment:
```suggestion
import { mockStore } from '@deriv/stores';
```
@rostislav-deriv Use `TRootStore` instead 🙏🏻